heartpoints-k8s.yml

```diff
@@ -34,10 +34,31 @@ spec:
           initialDelaySeconds: 10
           periodSeconds: 10
 ---
+apiVersion: networking.gke.io/v1beta1
+kind: ManagedCertificate
+metadata:
+  name: heartpoints-org-cert
+spec:
+  domains:
+    - heartpoints.org
+---
+apiVersion: networking.gke.io/v1beta1
+kind: ManagedCertificate
+metadata:
+  name: www-heartpoints-org-cert
+spec:
+  domains:
+    - www.heartpoints.org
+---
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
   name: heartpoints-ingress
+  annotations:
+    kubernetes.io/ingress.global-static-ip-name: heartpoints-org
```

Comment:
the name of the [static IP of the LB](https://console.cloud.google.com/networking/addresses/list?project=heartpoints-org&addressesTablesize=50) this static ip was created manually, I updated our readme to better explain how it was done.